cli/internal/cmd/manifestUpdate.go

```diff
@@ -145,6 +150,11 @@ func runUpdateApply(cmd *cobra.Command, args []string) error {
 		)
 	}
 	cmd.Println(msg)
+
+	if len(recoveryData) > 0 {
+		return saveNewRecoveryData(cmd, recoveryData)
```

Comment:
if this returns an err, we should print the raw recoveryData map so that it doesn't get lost